admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/ApplicationConfigServiceImpl.java

```diff
@@ -52,7 +52,11 @@ public ApplicationConfigResponseDto getLanguageConfigDetails() {
 		Map<String, String> response = new HashMap<String, String>();
 		String[] arrayOfKeys = uiConfigs.split(";");
 		for (String key : arrayOfKeys) {
-			response.put(key.split(":")[0], key.split(":")[1]);
+			if (key.split(":").length > 1) {
```

Comment:
Why do you need a else? Can you not just set to null and overwrite if length > 1 that makes the code more readable.

Reply:
taken care